src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,189 @@
+package duke.parser;
+
+import duke.command.*;
+import duke.exception.AdditionalException;
+
+/**
+ * Represents a command parser that deals with making sense of the user command.
+ */
+public class Parser {
+
+    /**
+     * This is the parse method that takes in the fullCommand and makes sense of what type of command it is.
+     * It then creates a Command object, depending on the type of command, and returns in to the caller.
+     *
+     * @param fullCommand The user input.
+     * @return The Command object, depending on the type of the command.
+     * @throws AdditionalException If the command is invalid.
+     * @see AdditionalException
+     */
+    public static Command parse(String fullCommand) throws AdditionalException {
+        String[] words = fullCommand.split(" ");
+        String command = words[0].toLowerCase();
+        Command c;
```

Comment:
Alphabets are typically used to name iterators. In this case, it is better to [name with a noun](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions).